README.md

```diff
@@ -57,7 +57,7 @@ Now that you have access to the `h1#header` element, use Javascript to change th
 
 **YOUR NOTES**
 ```
-
+I guess I have to stay in the window.onload function
```

Comment:
You can define other functions outside of there, but call them from inside there.